CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+<a name="0.4.3"></a>
+## 0.4.3 (2021-12-06)
```

Comment:
This should be 0.4.4. Also the 1st and 3rd items down below were already included in 0.4.3.